beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -477,6 +482,39 @@ impl DataColumnsByRootRequest {
     }
 }
 
+/// Request a number of beacon data columns from a peer.
+#[derive(Encode, Decode, Clone, Debug, PartialEq)]
+pub struct LightClientUpdatesByRangeRequest {
+    /// The starting period to request light client updates.
+    pub start_period: u64,
+    /// The number of periods from `start_period`.
+    pub count: u64,
+}
+
+impl LightClientUpdatesByRangeRequest {
+    pub fn max_requested<E: EthSpec>(&self) -> u64 {
```

Comment:
we don't need the generic parametrization here right?

Reply:
thanks, i had meant to move this value to `EthSpec`

[5928632](https://github.com/sigp/lighthouse/pull/6383/commits/59286323226c11154bebcc011545015d23c60d5e)

Reply:
no worries Eitan, but can we remove entirely the `EthSpec` dependency from this structure and not have this function here? See https://github.com/sigp/lighthouse/pull/6462#discussion_r1792315770 for the why.

Reply:
ah ok makes sense. Ive removed the ethspec dependency

0798629